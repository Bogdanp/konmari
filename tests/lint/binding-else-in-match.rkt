#lang racket/base

(require racket/match)

(match 10
  [x x]
  [else 'foo])
