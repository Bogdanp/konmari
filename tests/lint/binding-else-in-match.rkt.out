/binding-else-in-match.rkt:7:4:error:use _ instead of else in the fallthrough case of a match expression
